src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java

```diff
@@ -162,7 +162,7 @@ public EditorForm getEditForm(Locale uiLocale, Locale locale, String uuidOrPath)
                     extendContext.put(contextEntry.getKey(), contextEntry.getValue());
                 }
 
-                return getValueConstraints(nodeTypeRegistry.getNodeType(primaryNodeType), editorFormField, node, parentNode, locale, extendContext);
+                return getValueConstraints(nodeTypeRegistry.getNodeType(primaryNodeType), editorFormField, node, locale, extendContext);
```

Comment:
since we're removing parentNode, maybe we can clean up this line as well: https://github.com/Jahia/content-editor/blob/c2f6aec09a54eb99db4ee768b375885ea2e9af7a/src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java#L152

but I also have a question about removing parent node context below as well so this could also be impacted

Reply:
We can't, it's used by generateEditorFormField method below